docs/pbc/all/payment-service-providers/payone/integrate-payone.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Payone
+description: Learn how you can integrate the Payone app into your Spryker shop
+template: howto-guide-template
+---
+
+To integrate Payone, follow these steps:
+
+1. In your store's Back Office, go to **Apps&nbsp;<span aria-label="and then">></span> Catalog**.
+2. Click **Payone**.
+   This takes you to the Payone app details page.
```

Comment:
Wrap page names in bold

Reply:
this is just how we call the page, analogously to "product details page".